docs/general/administration/hardware-selection.md

```diff
@@ -25,6 +25,12 @@ These specs are the **recommended** specs to run Jellyfin. They are not minimum
 
 :::
 
+:::note AMD Ryzen CPUs and Hardware Encoding
+
+AMD Ryzen CPUs are often percieved to have inferior hardware encoding quality. While this is true when using integrated graphics for encoding, They will provide the same encoding quality and performance as Intel when encoding on a discrete GPU.
```

Comment:
In my opinion, if we're going to say this bold statement, I'd link to reputable sources.

If they don't exist, I'd rather use a more vague language.

Reply:
This is from my limited testing. However since I don't know of any reputable sources to link to I'm making the wording more vague.

Reply:
I know, and it's not the first time I've heard about it too ^^.

However, it's all "rumors" and we don't know if it's particular to certain hardware, if next family of graphics it will be fixed, etc etc... So better stay safe than sorry imo!